Vagrantfile

```diff
@@ -73,4 +73,5 @@ Vagrant.configure("2") do |config|
   #   apt-get update
   #   apt-get install -y apache2
   # SHELL
+  config.vm.provision "shell", path: "setup.sh", privileged: false
```

Comment:
@Yoavsc129 I think the name of the file should be "bootstrap.sh"

Reply:
Does it matter? It matches the name of the setup file